content/drone-plugins/drone-s3/index.md

```diff
@@ -72,6 +86,14 @@ pipeline:
 +   endpoint: https://play.minio.io:9000
 ```
 
+# Secret Reference
+
+aws_access_key_id
```

Comment:
Could you add the other possible variable names as well, please? Same for the other secret. It should be just a comma separated string.

Reply:
like that ?

Reply:
Just checked the source, sorry... Your first commit was right. 

But there are other secrets available like s3_region, s3_endpoint and s3_bucket.